scripts/entrypoint.sh

```diff
@@ -18,6 +18,7 @@ GTP_EXTENSION_HEADER_PRESENT=${GTP_EXTENSION_HEADER_PRESENT:-no}
 ENABLE_5G_FEATURES=${ENABLE_5G_FEATURES:-no}
 REGISTER_NRF=${REGISTER_NRF:-no}
 USE_FQDN_NRF=${USE_FQDN_NRF:-no}
+SPGWC0_IP_ADDRESS=${SPGWC0_IP_ADDRESS:-127.0.0.1}
```

Comment:
@orion-belt 

can you make it conditional like you did for the 5G components?

It should be based on the `ENABLE_5G_FEATURES` status.

if `no`, then your default value is good.

It will also retrigger CI since it failed for something not related to your work.

Raphael